docs/install.md

```diff
@@ -879,6 +883,13 @@ The possible `phase` values and their meaning are as below:
 <sup>1</sup> This check has not been implemented yet. Instead, the deployment goes straight to `Running` after creating sub-resources.
 <sup>2</sup> Failure reason is supposed to be carried by one of additional `DeploymentStatus` field, but not implemented yet.
 
+#### Deployment Events
+
+The PMEM-CSI operator posts events on the progression of a `Deployment`. In case,
```

Comment:
s/progression/progress/

"If the deployment is in the `Failed` state, then one can...."

Reply:
Done